2020-03-26 14:11:26,480 INFO: Microblog startup [in C:\Users\sgoli\Documents\flask-app\app\__init__.py:51]
2020-03-26 14:59:51,806 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 15:32:47,408 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 15:33:13,639 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 15:38:47,431 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 15:40:43,810 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 15:42:39,939 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 15:43:24,355 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:07:08,021 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:08:12,264 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:08:39,508 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:13:54,764 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:18:44,762 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:32:47,097 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:37:18,209 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:37:42,154 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:41:10,766 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:46:53,600 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:47:13,893 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:47:27,803 ERROR: Exception on /Inventory [GET] [in c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\sgoli\Documents\GitHub\flask-app\app\routes.py", line 48, in inventory
    bourbonForm = BourbonForm()
NameError: name 'BourbonForm' is not defined
2020-03-26 16:48:13,054 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:49:29,986 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:54:23,492 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:54:32,640 ERROR: Exception on /Inventory [GET] [in c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
TypeError: inventory() missing 1 required positional argument: 'filter'
2020-03-26 16:54:55,614 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:56:21,688 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:57:55,080 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:58:00,406 ERROR: Exception on /Inventory [GET] [in c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2130, in make_response
    " {rv.__class__.__name__}.".format(rv=rv)
TypeError: The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a int.
2020-03-26 16:58:37,632 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 16:58:46,345 ERROR: Exception on /Inventory [GET] [in c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2130, in make_response
    " {rv.__class__.__name__}.".format(rv=rv)
TypeError: The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a int.
2020-03-26 16:59:18,651 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 17:00:15,607 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 17:02:58,718 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-26 17:04:42,410 ERROR: Exception on /Inventory [GET] [in c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    return self.finalize_request(rv)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 1967, in finalize_request
    response = self.make_response(rv)
  File "c:\users\sgoli\documents\flask-app\venv\lib\site-packages\flask\app.py", line 2130, in make_response
    " {rv.__class__.__name__}.".format(rv=rv)
TypeError: The view function did not return a valid response. The return type must be a string, dict, tuple, Response instance, or WSGI callable, but it was a int.
2020-03-26 17:07:01,702 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-27 08:07:19,986 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-27 08:13:35,130 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-27 08:14:35,455 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-27 08:15:38,824 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
2020-03-27 08:15:47,027 INFO: Microblog startup [in C:\Users\sgoli\Documents\GitHub\flask-app\app\__init__.py:51]
